CIP-optional-drep-compensation/README.md

```diff
@@ -0,0 +1,57 @@
+---
+CIP: CIP-DRep-Comp
+Title: Optional DRep Compensation
+Category: Metadata
+Status: Proposed
+Authors:
+    - Philip DiSarro <info@anastasialabs.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/991
+Created: 2025-02-19
+License: CC-BY-4.0
+---
+
+## Abstract
+<!-- A short (\~200 word) description of the proposed solution and the technical issue being addressed. -->
+We propose an opt-in standard for DRep compensation that does not require any changes to the ledger. When delegating to a DRep a user can opt-in to donate a percentage of their staking rewards to their DRep. 
+
+## Motivation: why is this CIP necessary?
+Introducing DRep compensation into the ledger will be a an absolutely massive undertaking, that will involve a large amount of engineering hours and a non-trivial price-tag. Also, the community has not yet reached consensus on whether DRep compensation is even desirable, or should be mandatory, or what the compensation scheme would even look like.
+
+The best way to approach feature development is to release an MVP, evaluate its reception in the market, and use that data to determine the best way to proceed. This also allows us to introduce the feature without imposing it on anyone, it is opt-in, if you do not wish to compensate your DRep you can simply not opt-in. If, at any time, you wish to adjust the percentage of rewards that you are donating to your DRep you can just issue a new delegation certificate and adjust the amount or opt-out entirely.
+
+## Specification
+This feature is extremely simple to implement. When a user delegates to a DRep, the user should receive a prompt asking if they want to donate a percentage of their staking rewards to that DRep (ie. if they want to opt-in to compensate their DRep).
+
+If they choose to opt-in, they enter the percentage of their staking rewards that they would like their DRep to receive. The DRep delegation transaction will include that percentage in the transaction metadata in the following format:
```

Comment:
```suggestion
If they choose to opt-in, they enter the percentage of their staking rewards that they would like their DRep to receive. The DRep delegation transaction will include that percentage in the transaction metadata, with metadatum label `3692`, in   the following format:
```
add specific detail on `metadatum_label`, this PR should also this label to the CIP-10 registry to reserve it